spec/acceptance/reboot_finished_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'spec_helper_acceptance'
+
+describe 'Reboot Module - Reboot when Finished' do
```

Comment:
This test-suite is already acting in the context of the `Reboot Module`, no need to repeat yourself here.